.github/workflows/run-tests.yml

```diff
@@ -33,7 +33,8 @@ jobs:
   test:
     runs-on: ${{ inputs.os }}
     timeout-minutes: 60
-    name: ${{ inputs.os }}
+    name: ${{ inputs.testShortName }}
+    if: inputs.os != 'windows-latest' || inputs.testShortName != 'Playground'
```

Comment:
Where were the "playground" tests filtered out previously? Can that be removed now that we are filtering them out here?

Another option could be to update the playground tests to have `[RequiresDocker]` on them, and just always run them, even if they no-op for now.

Reply:
They are controlled from the project file:
https://github.com/dotnet/aspire/blob/0574cfcd61a28a538acca941bec71880e3b7f95f/tests/Aspire.Playground.Tests/Aspire.Playground.Tests.csproj#L9-L13

I'll move to using the attribute instead of a condition here.